README.md

```diff
@@ -24,8 +24,17 @@ cd osier
 pip install .
 # to also install the documentation dependencies
 pip install .[doc]
+
+# followed by 
+pip install pyomo==6.4.1
 ```
 
+> [!NOTE]
+> Although `pyomo` is a dependency, the current version of `pyomo` (6.7.1, as of 2/29/24) has a bug
+that prints erroneous errors during an `osier` simulation. Therefore, users are recommended to 
+install a specific version of `pyomo` after the main installation of `osier`. There is an open issue [#50](https://github.com/arfc/osier/issues/50) 
+related to this concern.
```

Comment:
Perhaps you should specify the maximum `pyomo` version we need in `setup.py` that doesn't have the bug?

Reply:
I don't know what that version is without a lot more testing and I haven't the patience for it right now. I'm also having trouble specifying a version in the `setup.py` file. See #50